src/main/java/pl/cleankod/exchange/core/domain/Account.java

```diff
@@ -1,13 +1,15 @@
 package pl.cleankod.exchange.core.domain;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
```

Comment:
This was the easiest way to get rid of the 'redundant' _value_ in json responses, but it breaks the ideology of not including framework/library specific things in application core